src/v/storage/tests/appender_chunk_manipulations.cc

```diff
@@ -53,7 +53,7 @@ SEASTAR_THREAD_TEST_CASE(chunk_manipulation) {
         // 10 bytes on the next page
         BOOST_REQUIRE_EQUAL(c.flushed_pos() % alignment, 10);
         BOOST_TEST_MESSAGE("10 bytes spill over: " << c);
-        c.append(b.data() + i, alignment + 10);
+        c.append(b.data() + i, alignment() + 10);
```

Comment:
This would probably be easier to parse if the type was named `alignment_t`.

Reply:
I think the precedent is set for not using _t most places we do strong types for scalars, e.g. it's model::offset rather than model::offset_t

Reply:
yeh. i usually don't say anything if folks use `_t` but i don't think we have many such cases. maybe a good thing for the languishing style guide i started :/